### PR TITLE
[WIP] API consistency review

### PR DESCRIPTION
**Pull request by fredrikekre**
Trying to make the API a bit more consistent, in particular:

TODOs:
- [ ] Use same naming for getters (`ndofs` / `getndofs`)
- [ ] Remove unused/useless exports
- [ ] ...

### CODE REVIEW

File: src/Export/VTK.jl [hunk 1]
```diff
@@ -1,134 +1,47 @@
...
-```
-
-**Details**
-
-This is a thin wrapper around the function `vtk_grid` from the [`WriteVTK`](https://github.com/jipolanco/WriteVTK.jl) package.
-
-For information how to add cell data and point data to the resulting VTK object as well as how to write it to a file see
-[https://github.com/jipolanco/WriteVTK.jl#generating-an-unstructured-vtk-file](https://github.com/jipolanco/WriteVTK.jl#generating-an-unstructured-vtk-file)
 """
-function vtk_grid{dim,T}(filename::AbstractString, coords::Vector{Vec{dim,T}}, topology::Matrix{Int}, celltype::VTKCellTypes.VTKCellType)
```
Comment (fredrikekre) [inline src/Export/VTK.jl]: OK to remove this? It feels superseeded by `vtk_grid(filename, grid::Grid)`, at least I don't use it.

File: src/Dofs/DofHandler.jl [hunk 2]
```diff
@@ -230,16 +227,10 @@ function _create_sparsity_pattern(dh::DofHandler, sym::Bool)
 end
 
 
-vtk_grid(filename::AbstractString, dh::DofHandler) = vtk_grid(filename, dh.grid)
+WriteVTK.vtk_grid(filename::AbstractString, dh::DofHandler) = vtk_grid(filename, dh.grid)
 
 # Exports the FE field `u` to `vtkfile`
-function vtk_grid(filename::AbstractString, dh::DofHandler, u::Vector)
```
Comment (fredrikekre) [inline src/Dofs/DofHandler.jl]: This felt wrong for `vtk_grid` so I removed it, instead do

```julia
vtkfile = vtk_grid("filename", dh)
vtk_point_data(vtkfile, dh, u)
```

which is a bit more transparent, ok? :p

File: src/Export/VTK.jl (hunk 1 above)
Comment (KristofferC) [inline src/Export/VTK.jl]: Sure

File: src/Dofs/DofHandler.jl (hunk 2 above)
Comment (KristofferC) [inline src/Dofs/DofHandler.jl]: Oki

**Pull request merged**